dojo/tools/noseyparker/parser.py

```diff
@@ -17,85 +17,158 @@ def get_label_for_scan_types(self, scan_type):
 
     def get_description_for_scan_types(self, scan_type):
         return "Nosey Parker report file can be imported in JSON Lines format (option --jsonl). " \
-               "Supports v0.16.0 of https://github.com/praetorian-inc/noseyparker"
+               "Supports v0.16.0 and v0.22.0 of https://github.com/praetorian-inc/noseyparker"
```

Comment:
Hi @Himan10 ,
I suggest to close this PR and just review mine. This PR also targets branch master which is not the right target.

Reply:
okay but could you give me permissions to push changes to your PR ? It looks like I don't have permissions. I reviewed your changes tho but it was failing with scans where `--git-history` was set to none.

Reply:
just add a comment in the code inside the PR and submit the review. (Do a code review)

Reply:
@manuel-sommer :+1: on keeping the original #11565 open and closing this one